src/Command/Argument/Search/ExplainArguments.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2023 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Command\Argument\Search;
+
+class ExplainArguments extends CommonArguments
```

Comment:
why this is needed if its empty?

Reply:
@shacharPash We agreed to provide separate interface for command arguments if it's needed, well `CommonArguments` is an interface that contain arguments to be used across multiple commands, however this command can accept only arguments already specified within `CommonArguments`, so this object was created just to separate interfaces. In future if it could accept more specific arguments, it will be specified here